### PR TITLE
Changed Bretonia Gunboat, Destroyer, & Battleship

### PR DESCRIPTION
**Pull request by FireFlyEx**
Here is the first set of ships to be updated:

**New**
```
                                 Angular & Linear Speeds
Bretonian Battleship   L/R: 6;  U/D: 4;  Roll: 16.87; Forward: 80
Bretonian Destroyer    L/R: 8;  U/D: 4;  Roll: 16.08; Forward: 100
Bretonian Gunboat      L/R: 30; U/D: 24; Roll: 60.09; Forward: 120
```
**Old**
```
                                 Angular & Linear Speeds
Bretonian Battleship   L/R: 6.52;  U/D: 1.63;  Roll: 6.52;  Forward: 80
Bretonian Destroyer    L/R: 8.68;  U/D: 3.47;  Roll: 8.68;  Forward: 100
Bretonian Gunboat      L/R: 28.65; U/D: 28.65; Roll: 28.65; Forward: 120
```
_L/R = Left/Right
U/D = Up/Down_

Modified shiparch entries, engine entries, and appropriate cockpit files.

### CODE REVIEW

**Comment by erikroe** [comment]
I'll be checking them out in a moment. Though, the speeds of the ships were intentionally set beforehand on what they were to support their roles in combat. Please change those back to the original values.

**Comment by erikroe** [comment]
Also on the Linear Drag; For the BR Gunboat it was intentional to have a portion of it to be set on the engine and one portion on the ship. The reason for this: Reducing the effect of engine kill. Engine kill on capital ships is limited by setting Linear Drag on the ship instead of the engine. This prevents them from using it, effectively. For the big guns to not have suddenly a larger range when dropping from cruise by using engine kill. For Gunboats this however was partially allowed. That is why their linear drag values were distributed on ship and engine. I would like to see that behaviour being kept.

**Comment by erikroe** [comment]
I tested the ships now. The Bretonian Gunboat was far too much nerfed. It's role as anti-fighter/freighter support is impossible to fulfill now with this far lower turning rates.

**Comment by erikroe** [comment]
Tested with the adjusted BR Gunboat settings again. It feels quite odd from before. The movement appears stiff. Like you said, more like a heavy fighter. I think it has to do with the camera settings you applied to it.
I have one question though: Why was the camera movements and all in the shiparch even changed? I thought it to be just good as it was before. Same as the steering speed really.

File: DATA/SHIPS/shiparch.ini [hunk 1]
```diff
@@ -5628,7 +5629,7 @@ DA_archetype = ships\bretonia\br_destroyer\br_destroyer.cmp
 material_library = ships\bretonia\br_capships.mat
 material_library = fx\envmapbasic.mat
 envmap_material = envmapbasic
-linear_drag = 10000
+linear_drag = 24000
```
Comment (erikroe) [inline DATA/SHIPS/shiparch.ini:5632]: The increased linear drag here and at the battleship reduce the feeling of them being heavy. They break too fast, taking away the illusion of inertia.

**Review by erikroe — CHANGES_REQUESTED** [review]

**Comment by FireFlyEx** [comment]
> 

One of the things I am doing is normalizing the data for capital ships. Changing the camera settings gives more weight to the ship and less wild turning in the camera view, which tends to be less disorienting to the player and helps those who suffer from motion sickness (like me). I can find a medium point if you like? Or I can forget about all of this stuff since you don't see any of it as beneficial. I can work on something else.

**Pull request closed without merging**